Composer/packages/ui-plugins/composer/src/defaultRecognizerSchema.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { RecognizerUISchema, FallbackRecognizerKey, RecognizerOptions } from '@bfc/extension-client';
+import { SDKKinds } from '@bfc/shared';
+import formatMessage from 'format-message';
+import { RegexIntentField, CustomRecognizerField } from '@bfc/adaptive-form';
+
+const FallbackRecognizerJsonEditor: RecognizerOptions = {
+  displayName: formatMessage('Custom recognizer'),
```

Comment:
I saw the field `displayName` is typed with `UIOptionValue<string>` which is expanded as `string | (data: any) => string`.
So for those cases without input parameters, I just wrote them as a plain string.

Reply:
You misunderstand me. The way `formatMessage` works is that it will translate the string when invoked. If this is defined as a module constant, it will only be invoked when the module is loaded. This will not respond to changes in the locale settings. It needs to be wrapped in a function in order to get the latest locale setting.

Reply:
I see. I updated all `displayName` appearances and append it to the comment.